src/main/java/org/folio/auth/authtokenmodule/apis/RouteApi.java

```diff
@@ -331,6 +331,7 @@ private void handleSignLegacyToken(RoutingContext ctx) {
         var message = "Tenant not a legacy token tenant as specified in this module's environment or system " +
           "property. Cannot issue non-expiring legacy token.";
        endText(ctx, 404, new LegacyTokenTenantException(message));
+       return;
```

Comment:
There's a unit test for the 404 response [here](https://github.com/folio-org/mod-authtoken/blob/37d85bca2a9edf2523a6845f4131f399c84506fb/src/test/java/org/folio/auth/authtokenmodule/AuthTokenTest.java#L699). The reason this issue was missed is that it responds `404` to clients and tests, but the response continues after `endText` because of the lack of the `return`. This produces an error in vertx (I believe) which gets logged.